### PR TITLE
JmeterNamesTemplater v1.1 and UtilityJmeterFunctions v1.1.3

### PR DESCRIPTION
**Pull request by DanilGolikov**
Created [JmeterNamesTemplater](https://github.com/DanilGolikov/JmeterNamesTemplater) that allows you to rename items according to the config file

### CODE REVIEW

File: site/dat/repo/various.json [hunk 1]
```diff
@@ -2099,5 +2099,29 @@
...
+  },
+  {
+    "id": "JmeterNamesTemplater",
+    "name": "JMeter Names Templater",
+    "description": "Jmeter Names Templater is a plugin for Jmeter that allows you to set naming templates for elements and rename them based on a described configuration file.",
+    "vendor": "Daniil Golikov",
+    "markerClass": "com.example.jmeter.plugin.gui.RenameElementsMenuCreator",
+    "helpUrl": "https://github.com/DanilGolikov/JmeterNamesTemplater/blob/master/README.md",
+    "componentClasses": [
+      "com.example.jmeter.plugin.gui.RenameElementsMenuCreator",
```
Comment (undera) [inline site/dat/repo/various.json:2111]: `com.example.jmeter.plugin` does not look like a good namespace for plugin. Can you come up with something more distinctive? Something more author-specific.
Should be a simple refactoring.

File: site/dat/repo/various.json [hunk 2]
```diff
@@ -2099,5 +2099,29 @@
...
       }
     }
+  },
+  {
+    "id": "JmeterNamesTemplater",
+    "name": "JMeter Names Templater",
+    "description": "Jmeter Names Templater is a plugin for Jmeter that allows you to set naming templates for elements and rename them based on a described configuration file.",
+    "vendor": "Daniil Golikov",
+    "markerClass": "com.example.jmeter.plugin.gui.RenameElementsMenuCreator",
+    "helpUrl": "https://github.com/DanilGolikov/JmeterNamesTemplater/blob/master/README.md",
```
Comment (undera) [inline site/dat/repo/various.json:2109]: `https://github.com/DanilGolikov/Custom_jmeter_functions` should be simpler and sufficient URL here.

Also I'd work on that page for better presentation, titles and help document structure. Up to you, though.

**Review by undera — CHANGES_REQUESTED** [review]

**Comment by DanilGolikov** [comment]
Hi! I corrected the namespace, I didn't pay attention to the fact that it looks stupid xD before.
I also added another plugin (CustomJmeterFunctions), with your recommendations

**Review by DanilGolikov — COMMENTED** [review]
.

File: site/dat/repo/various.json [hunk 3]
```diff
@@ -2099,5 +2099,47 @@
...
+        "downloadUrl": "https://github.com/DanilGolikov/JmeterNamesTemplater/releases/download/v1.1-yaml/jmeter-plugins-names-templater-1.1.jar",
+        "libs": {
+          "snakeyaml": "https://search.maven.org/remotecontent?filepath=org/yaml/snakeyaml/2.2/snakeyaml-2.2.jar"
+        }
+      }
+    }
+  },
+  {
+    "id": "CustomJmeterFunctions",
+    "name": "Custom Jmeter Functions",
```
Comment (undera) [inline site/dat/repo/various.json:2129]: There is existing plugin with this name, you need to choose a different name. Preferably a name that explains what kind of functions are inside. Like "Functions for strings templating" or like that.
Don't forget to change the ID of plugin, and also the description of it should explain more what is inside.

**Comment by undera** [comment]
Please make the title of PR to reflect what is inside. Look at other PRs to get the idea

**Review by undera — CHANGES_REQUESTED** [review]
Please address my comments

**Comment by undera** [comment]
Looks like your JSON violate descriptor schema: https://github.com/undera/jmeter-plugins/actions/runs/10546658418/job/29218873882?pr=665

**Comment by undera** [comment]
Looks good now

**Pull request merged**